### ISSUE TITLE
[doc] Update instructions for wheel installation

### ISSUE THREAD

**Issue by bushshrub** [ray-project/ray#24533]
### What happened + What you expected to happen

https://docs.ray.io/en/master/ray-contribute/development.html#building-ray

The wheel installation link is a bit misleading since it only applies for Python 3.8 on linux. It would be best to remove the wheel link and replace it with a link directly to the nightly wheels.

### Versions / Dependencies

None

### Reproduction script

https://docs.ray.io/en/master/ray-contribute/development.html#building-ray

### Issue Severity

Low: It annoys or frustrates me.

**Comment by jianoaix** [comment]
That link is an example. The link to those 12 nightly wheels (OS * Python combinations) is already there, right?

**Comment by xiurobert** [comment]
Yup it is, just thought it might be a bit misleading. It does throw an error pretty quick if your system is incompatible.

**Comment by maxpumperla** [comment]
@xiurobert @jianoaix we could introduce code tabs for all options

**Comment by xiurobert** [comment]
Alternatively an easier solution would be to delete direct link to the wheel.

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
